hpedockerplugin/hpe_storage_api.py

```diff
@@ -105,11 +105,17 @@ def __init__(self, reactor, all_configs):
             all_configs=all_configs)
 
     def is_backend_initialized(self, backend_name):
+        if (backend_name not in self._backend_configs and
```

Comment:
Is this check really needed? if next 2 if conditions are failed we anyway return FAILED state?